### PR TITLE
Large data asset policy text

### LINKED ISSUE #1570: Large Assets and GitHub
I was emailing with @svmelton and @mdlincoln about an editorial issue on the English team, and it seemed like something to raise to the group here for comment. An upcoming lesson in the pipeline has an asset - a CSV file - that is necessary for the analysis, but the asset is large. It's about 130 megabytes, and GitHub is complaining when they try to upload it. The question is what to do with such things. 

@mdlincoln had good thoughts - that the best solution would be to make sure that datasets or assets like this go into a scholarly repository so they have a persistent link. He suggested using Zenodo, since we already use that for archiving the project. Either the author or the lesson editor could upload to it, and then we link to the DOI that is generated. Posting here for thoughts.

### PR DESCRIPTION
**Pull request by walshbr**
Closes #1570 re: large asset policy and GitHub/Zenodo.

Short paragraph that needs French and Spanish translation.

### CODE REVIEW

**Comment by fdlaramee** [comment]
Added French translation to consignes-redacteurs.md in the large-assets branch.

**Review by mdlincoln — DISMISSED** [review]

**Comment by jenniferisasi** [comment]
I'm working on the translation of this paragraph and ... surprise ... I'm going to raise an issue that might require just a few more words on our part. Zenodo is only available in English, do you think it is ok to say: 
> El/La editor/a debe asegurarse de que el/la autor/a de la lección puede navegar la interfaz de Zenodo, unicamente disponible en inglés, y ofrecerle ayuda en caso de no saber el idioma.

> The editor shall make sure that the author of the lesson can navigate Zenodo's interface, available in English only, offering their help if they don't know the language. 

@fdlaramee this is something you may want to add in your version too.

**Comment by walshbr** [comment]
@jenniferisasi great catch - that is an important change. Thank you!

**Comment by walshbr** [comment]
Actually - hold on further translation for now. I ran into a snag trying to implement Zenodo for large assets for the lesson I'm working on, and the text here might need one or two more sentences about how to upload things to get them to work successfully. More soon!

**Comment by mdlincoln** [comment]
FYI that "some checks haven't completed yet" status _ought_ to go away when you push a new commit to this PR (re #1604) if they don't and block merging, please make sure to ping me.

**Comment by walshbr** [comment]
Thanks for your patience! Was just chatting with @mdlincoln and have the finalized text for you. The extra piece that needs translating into French and Spanish is this:

>   When uploading to Zenodo, all files (even if there is a single file) should be compressed into a single zip file. The zip file should have the same slug used for the lesson file. This is only necessary when the total size of the assets for the lesson is larger than 25MB.

Thank you @programminghistorian/french-team and @programminghistorian/spanish-team! Sorry for the confusion and extra work.

File: en/editor-guidelines.md [hunk 1]
```diff
@@ -182,6 +182,10 @@ Full instructions on adding images is available in [Author Submission Instructio
...
 
  - `/assets/LESSON-SLUG/Louvre-Paintings-1.csv`
+ 
+ Occasionally, large assets used by particular lessons might be too large to be stored in our GitHub repository. If this is the case, we recommend that authors upload their assets to [Zenodo](https://zenodo.org/) for archiving and then provide the lesson editor with the DOI generated by Zenodo for linking from within the lesson. In the event that such datasets might already exist in an institutional repository, we still recommend uploading the version of the dataset used in The Programming Historian to Zenodo for the purposes of consistent use across all of our lessons.
+ 
+ When uploading to Zenodo, all files (even if there is a single file) should be compressed into a single zip file. The zip file should have the same slug used for the lesson file. This is only necessary when any individual asset for the lesson is larger than 25MB.
```
Comment (mdlincoln) [inline en/editor-guidelines.md]: Ah, I'd say "necessary when the **total** size of the assets for a lesson is larger than 25MB"

**Review by mdlincoln — CHANGES_REQUESTED** [review]

File: en/editor-guidelines.md (hunk 1 above)
Comment (walshbr) [reply en/editor-guidelines.md]: Fixed!

**Review by rivaquiroga — DISMISSED** [review]
In the Spanish translation I put the last sentence of the paragraph at the beginning (_When the total size..._).  Seems clearer to know first that +25MB is the only scenario were compression is needed, and then read the explanation.

**Comment by fdlaramee** [comment]
Added the necessary language to the French version. Should be good to go, I hope!

**Review by fdlaramee — APPROVED** [review]

**Review by mdlincoln — APPROVED** [review]

**Comment by walshbr** [comment]
Thank you! Should be good for merging now then.

**Pull request merged**